doc/stages/readers.hdf.rst

```diff
@@ -0,0 +1,101 @@
+.. _readers.hdf:
+
+readers.hdf
+===============
+
+The **HDF reader** reads from files in the HDF5 format. See the
```

Comment:
The **HDF reader** reads data from files in the (link) HDF5 format. You must explicitly specify a mapping of HDF datasets to PDAL dimensions using the ``dimensions`` parameter. ALL dimensions must be scalars and be of the same length. Compound types are not supported at this time.